srml/bridge/Cargo.toml

```diff
@@ -8,9 +8,15 @@ edition = "2018"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false }
+client = { package = "substrate-client", path = "../../core/client" }
```

Comment:
You can't import `client`, `std` cannot be disabled.

Also, you can't import `state-machine` below.